web/src/pages/access/request/[id].tsx

```diff
@@ -389,12 +389,12 @@ const Home = () => {
                       {...register("reason", {
                         validate: (value) => {
                           const res: string[] = [];
```

Comment:
Just use the built in maxLength validation for this rather than the validate function